2024-12-21 18:52:21,928 | WARNING | Неверный тип данных для объекта Runner
Traceback (most recent call last):
  File "D:\!Мария\!УЧЕБА\02_URBAN\003_ПРОЕКТЫ\WORK_PROJECT_00\MODULES\module_12\test_12_4.py", line 33, in test_run
    run1 = rt.Runner(35)
           ^^^^^^^^^^^^^
  File "D:\!Мария\!УЧЕБА\02_URBAN\003_ПРОЕКТЫ\WORK_PROJECT_00\MODULES\module_12\rt_with_exceptions.py", line 6, in __init__
    raise TypeError(f'Имя может быть только строкой, передано {type(name).__name__}')
TypeError: Имя может быть только строкой, передано int
2024-12-21 18:52:21,930 | WARNING | Неверная скорость для Runner
Traceback (most recent call last):
  File "D:\!Мария\!УЧЕБА\02_URBAN\003_ПРОЕКТЫ\WORK_PROJECT_00\MODULES\module_12\test_12_4.py", line 19, in test_walk
    run2 = rt.Runner('Max', -3)
           ^^^^^^^^^^^^^^^^^^^^
  File "D:\!Мария\!УЧЕБА\02_URBAN\003_ПРОЕКТЫ\WORK_PROJECT_00\MODULES\module_12\rt_with_exceptions.py", line 11, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -3
